src/codeu/chat/client/simplegui/NewAccount.java

```diff
@@ -53,22 +62,25 @@ public NewAccount (JFrame mainFrame) {
         usernamePanel.add(userNameTextField);
         mainPanel.add(usernamePanel);
 
+        // TODO: 
         // password label and textfield
-        JPanel userPasswordPanel = new JPanel();
-        JLabel userPasswordLabel = new JLabel("Choose a password up to 10 characters");
-        userPasswordPanel.add(userPasswordLabel);
-        JPasswordField userpasswordTextField = new JPasswordField(10);
-        userPasswordPanel.add(userpasswordTextField);
-        mainPanel.add(userPasswordPanel);
+        // JPanel userPasswordPanel = new JPanel();
```

Comment:
Remove this altogether. Git has history, can always see past commit to get this back